chicago/permit_cleaning.py

```diff
@@ -2,7 +2,7 @@
 Chicago Permit Ingest Process - Automation
 
 This script automates the current process for cleaning permit data from the Chicago Data Portal's Building Permits table
-and preparing it for upload to iasWorld via SmartFile. This involves fetching the data, cleaning up certain fields, 
+and preparing it for upload to iasWorld via SmartFile. This involves fetching the data, cleaning up certain fields,
```

Comment:
The changes in this file are exclusively removing stray whitespace, which you can confirm by [hiding whitespace changes in the diff](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#diff-view-options). I did this accidentally by opening and saving the file in my editor, but I figure it's probably good to remove this stray whitespace so I left the diff in.